.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
yes, it is shared settings config to prevent the masses from putting spaces in my tabs